src/cli/project/version/bump.rs

```diff
@@ -0,0 +1,36 @@
+use crate::Project;
+use rattler_conda_types::VersionBumpType;
+
+pub async fn execute(mut project: Project, bump_type: VersionBumpType) -> miette::Result<()> {
+    // get version and exit with error if not found
+    let current_version = project
+        .version()
+        .as_ref()
+        .ok_or_else(|| {
+            eprintln!(
+                "{}No version found in manifest.",
+                console::style(console::Emoji("✖ ", "")).red(),
+            );
+        })
+        .unwrap()
+        .clone(); // NOTE(hadim): ideas are welcome to get rid of that clone.
```

Comment:
Or do we need that clone here?

Reply:
Im not behind a Pc to check, but why is the clone required?

Reply:
I'm getting an error below when setting the new version and saving it:

```
cannot borrow `project.manifest` as mutable because it is also borrowed as immutable
mutable borrow occurs here
```

Reply:
Right, in that case I think the clone is required.